malariagen_data/anopheles.py

```diff
@@ -248,6 +248,31 @@ class h12_params:
         The size of windows (number of SNPs) used to calculate statistics within.
         """,
     ]
+    cohort_size: TypeAlias = Annotated[
+        Optional[int],
+        """
+        Randomly down-sample to this value if the number of samples in the
+        cohort is greater. Raise an error if the number of samples is less
+        than this value.
+        """,
+    ]
```

Comment:
```suggestion
```

This parameter is already defined on the `base_params` class, don't think we need to duplicate here. Can just add the default values here.